auto_nag/topcrash.py

```diff
@@ -297,9 +297,8 @@ def handler(search_resp: dict, data: dict):
                 }
             )
 
-        signatures = ["=" + signature for signature in signatures]
-        chunks, size = NoCrashes.chunkify(signatures)
         signature_volume: dict = {signature: 0 for signature in signatures}
+        chunks, size = NoCrashes.chunkify(["=" + signature for signature in signatures])
```

Comment:
We are going to iterate twice on `signatures` this way, and could consume the iterator the first time.

Reply:
We either fix this by collecting the signatures in a list first, or we change the argument type to ensure it's something that can be iterated over multiple times